### ISSUE TITLE
authoringToolURI constraints

### ISSUE THREAD

**Issue by HaroldHallikainen** [SMPTE/st2098-2#5]
Issue 2: authoringToolURI constraints
ST 2098-2 requires that authoringToolURI be a "Uniform Resource Identifier as defined by IETF RFC 2396 with the authority component being a DNS resolvable domain name," but not all URIs have an authority component, e.g. URNs, relative URIs, etc.
It is not clear if ST 2098-2 means that authoringToolURI shall be:
•	any generic URI in absolute form, i.e. ://?; or
•	specifically HTTP URIs, i.e. http://example.com/hello
In other words, are the following valid authoringToolURI values?
https://example.com/hello
ftp://example.com/hello
urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6
Also, IETF RFC 2396 is obsoleted and should be replaced by RFC 3986.

**Comment by SteveLLamb** [comment]
Housekeeping for new revision: 
- This was completed in the 2022 revision, no longer applicable. 
- See https://doi.org/10.5594/SMPTE.ST2098-2.2022 Foreword.
